### ISSUE TITLE
Add popup action 

### ISSUE THREAD

**Issue by cyrilphoenix71** [piitaya/lovelace-mushroom#56]
It could be nice to have possibility to set pop-up action or press and long press with these popup right? 
![Screenshot_20220205-000626](https://user-images.githubusercontent.com/18350629/152789143-6bf8167d-45bf-4331-be75-ce704035b464.jpg)

**Comment by pierrecle** [comment]
That's a big thing that we already talk about with @piitaya, mainly because there's actually no such things in HASS, and how can we keep things simple in this case, etc. I'll close it just to don't keep in issues, but it's a `wontfix yet`.

**Comment by cyrilphoenix71** [comment]
maybe you can use milestones to store issue keep in mind for futur ?

**Comment by pierrecle** [comment]
TLDR; it's too early to think about milestone, let us release the first version, enjoy it and let's take a little rest while the community gives feedbacks :) So for now, I just close the issues that are in a too far future.

----

Not yet, our priority is to release the first non-beta version and fix coming issues on these cards (release/fix/enhance/new feature strategy). We released the beta version to get feedbacks, fix/enhance what we did, and start listening what are the first needs (not desires) of the community. We put a lot of effort and energy to reply to those needs and fixes quickly and releasing the first cards with more or less complex behaviours with the best UX we can provide (I let you see all the discussions and tries around light colors, rgb selector and coloring the slider/icon).

We don't know yet what will be in the next milestone, and for this particular thing, we don't know if we will provide bottom sheets or modals, as I said, nothing exists in HA and it's no a thing we can say "ok let's go" with the aim of mushroom I another issue (how display it, how to make it customizable, how to keep things easy for everyone etc).

@piitaya started this project few weeks ago, we didn't take the time to talk about milestones, I'm pretty sure he has no milestone plane in head. After our frustrations on minimalist-ui, we were just hurry to do it, enjoy it for ourselves, and share it with the community to see if there is needs from other people than us.

**Comment by cyrilphoenix71** [comment]
Thanks for take time to explain all of this. I completely agree with your vision, just may be little impatient :)
I juste take much interest of the project because I really thinks it will become a reference in HA world and for sure a day a part of HA.
I will certainly try to work on this project in near future.
I suggest this issue because I'm sure that in near future, popup will be introduce here or in HA like homekit
